tests/python/gpu/test_operator_gpu.py

```diff
@@ -2331,6 +2331,47 @@ def test_math():
             for op in ops:
                 run_math(op, shape, dtype, check_value=check_value)
 
+@with_seed()
+def test_np_nonzero():
+    class TestNonzero(HybridBlock):
+        def __init__(self):
+            super(TestNonzero, self).__init__()
+            
+        def hybrid_forward(self, F, x):
+            return F.npx.nonzero(x)
+
+    types = ['int32', 'int64', 'float64', 'float32', 'float16']
+    for hybridize in [True, False]:
+        for shape in [(),
+                      (1,),
+                      (1, 1),
+                      (1, 2, 3),
+                      (1, 0),
+                      (2, 0, 3)
+                      ]:
+            for oneType in types:
+                if oneType is 'float16':
+                    rtol=1e-2
+                    atol=1e-2
```

Comment:
I don't think this operator need different tolerance levels for different input data types.

Reply:
And no need to duplicate the same test in `test_operator_gpu.py`

Reply:
Done, thanks.